cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/resource/AbstractOperations.java

```diff
@@ -118,14 +116,23 @@ public AbstractOperations(Vertx vertx, C client, String resourceKind) {
     /**
      * Asynchronously patch the resource with the given {@code name} in the given {@code namespace}
      * with reflect the state given in the {@code patch}, returning a future for the outcome.
+     * The patch is applied to the related resource in cascade as well.
      * @param namespace The namespace of the resource to patch.
      * @param name The name of the resource to patch.
-     * @param patch The desired state of the resource..
+     * @param patch The desired state of the resource.
      */
     public Future<Void> patch(String namespace, String name, T patch) {
         return patch(namespace, name, true, patch);
     }
 
+    /**
+     * Asynchronously patch the resource with the given {@code name} in the given {@code namespace}
+     * with reflect the state given in the {@code patch}, returning a future for the outcome.
+     * @param namespace The namespace of the resource to patch.
+     * @param name  The name of the resource to patch.
```

Comment:
Two spaces here

Reply:
I tend to use tab between param name and description but it's fine using just space for me.